db/migrate/20180222113320_change_data_type_for_availability.rb

```diff
@@ -0,0 +1,10 @@
+class ChangeDataTypeForAvailability < ActiveRecord::Migration[5.1]
+
+  def change
+    add_column :products, :available, :boolean, null: false, default: true
+    Product.where(availability: 'available').find_each{|p| p.update_attributes(available: true)}
+    Product.where(availability: 'suspended').find_each{|p| p.update_attributes(available: false)}
+    remove_column :products, :availability, :boolean
+  end
```

Comment:
ActiveRecord won't be able to rollback the Product methods here. You need to use `reversible` instead:

http://edgeguides.rubyonrails.org/active_record_migrations.html#using-reversible